install/update.php

```diff
@@ -3933,29 +3943,6 @@ function do_networkport_migration($migration)
     );
     $migration->migrationOneTable($newTable);
 
-    // Update with mapping
```

Comment:
Alternative is to put back the `PluginGlpiinventoryMapping` class... Seems like this is useful only for very old versions of the plugin.

Reply:
It no longer exists on the 10bufxies branch; I can only find it on the glpi9.5 branch. In my opinion, it hasn’t been used for quite some time.